src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import './Navbar.css'
+
+export const Navbar = () => {
+  type StyleProp = {
+    isActive: boolean
+  }
+  type Style = {
+    color: string
+    textDecoration: string
+  }
+  const getStyle = ({ isActive }: StyleProp): Style => ({
+    color: isActive ? 'rgb(45, 125, 246)' : '',
+    textDecoration: isActive ? 'underline' : '',
+  })
```

Comment:
Recommend using a class name instead direct styles

Reply:
Made it class based.